backend/src/main/twirl/sbtBuild.scala.txt

```diff
@@ -0,0 +1,113 @@
+@(
+name: String,
+groupId: String,
+scalaVersion: String,
+httpDependencies: List[com.softwaremill.adopttapir.template.sbt.Dependency],
+monitoringDependencies: List[com.softwaremill.adopttapir.template.sbt.Dependency],
+jsonDependencies: List[com.softwaremill.adopttapir.template.sbt.Dependency],
+baseDependencies: List[com.softwaremill.adopttapir.template.sbt.Dependency],
+docDependencies: List[com.softwaremill.adopttapir.template.sbt.Dependency]
+)
+import com.softwaremill.SbtSoftwareMillCommon.commonSmlBuildSettings
+import sbt.Keys._
+import sbt._
+import sbtbuildinfo.BuildInfoKey.action
+import sbtbuildinfo.BuildInfoKeys.{buildInfoKeys, buildInfoOptions, buildInfoPackage}
+import sbtbuildinfo.{BuildInfoKey, BuildInfoOption}
+
+import scala.util.Try
+
+val httpDependencies = Seq(@for(dependency <- httpDependencies) {
+  @{dependency.asSbtDependency},}
+)
+@if(monitoringDependencies.nonEmpty) {
+val monitoringDependencies = Seq(@for(dependency <- monitoringDependencies) {
+  @{dependency.asSbtDependency},}
+)}
+@if(jsonDependencies.nonEmpty) {
+val jsonDependencies = Seq(@for(dependency <- jsonDependencies) {
+  @{dependency.asSbtDependency},}
+)}
+@if(baseDependencies.nonEmpty) {
+val baseDependencies = Seq(@for(dependency <- baseDependencies) {
+  @{dependency.asSbtDependency},}
+)}
+@if(docDependencies.nonEmpty) {
+val docDependencies = Seq(@for(dependency <- docDependencies) {
+  @{dependency.asSbtDependency},}
+)}
+val loggingDependencies = Seq(
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.4",
+  "ch.qos.logback" % "logback-classic" % "1.2.11",
+  "org.codehaus.janino" % "janino" % "3.1.7",
+  "de.siegmar" % "logback-gelf" % "4.0.2"
```

Comment:
a logback-gelf i janino sa tu potrzebne?

Reply:
Indeed I've copy-pasted them.
I will remove it.